Jenkinsfile

```diff
@@ -597,16 +599,8 @@ pipeline {
             deleteDir() /* clean up our workspace */
         }
         failure {
-            steps {
-                script {
-                    echo "The PR failed"
-                    
-                    if (IS_MASTER_BRANCH || IS_RELEASE_BRANCH)  {
-                        echo "Calling sendEmailNotification()"
-                        sendEmailNotification()
-                    }
-                }
-            }
+            echo 'The PR failed'
```

Comment:
A small thing, but this will print 'the PR failed' even if it's not a PR

Reply:
You have keen eyes. :) I wanted to keep irrelevant codes intact. I guess it's not required as it's obvious that it's failed - removed.